### ISSUE TITLE
请问可以分享一下残缺古诗语句预测的下游代码吗？为什么我这里显示的是随机的文字呢

### ISSUE THREAD

**Issue by WXuning** [Ethan-yt/guwenbert#21]
作者您好，真切的希望可以分享一下如何调用guwenbert实现对[MASK]预测的代码，我进行了一些尝试，但是输出的是随机的文字，而不是正确预测后的文字。万分感谢您。

**Comment by Ethan-yt** [comment]
发给你一个链接，里面的示例代码应该可以满足你的需求。参考以下代码，把roberta-base改成我的模型，把输入输出改成你自己的。
https://huggingface.co/docs/transformers/model_doc/roberta#transformers.RobertaForMaskedLM
```python
from transformers import AutoTokenizer, RobertaForMaskedLM
import torch

tokenizer = AutoTokenizer.from_pretrained("roberta-base")
model = RobertaForMaskedLM.from_pretrained("roberta-base")

inputs = tokenizer("The capital of France is <mask>.", return_tensors="pt")

with torch.no_grad():
    logits = model(**inputs).logits

# retrieve index of <mask>
mask_token_index = (inputs.input_ids == tokenizer.mask_token_id)[0].nonzero(as_tuple=True)[0]

predicted_token_id = logits[0, mask_token_index].argmax(axis=-1)
tokenizer.decode(predicted_token_id)
' Paris'

labels = tokenizer("The capital of France is Paris.", return_tensors="pt")["input_ids"]
# mask labels of non-<mask> tokens
labels = torch.where(inputs.input_ids == tokenizer.mask_token_id, labels, -100)

outputs = model(**inputs, labels=labels)
round(outputs.loss.item(), 2)
0.1
```
